defaults/network-ents_test.go

```diff
@@ -0,0 +1,157 @@
+package defaults
+
+import (
+	"syscall"
+	"testing"
+
+	"github.com/docker/libentitlement/secprofile"
+	"github.com/docker/libentitlement/types"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNetworkNoneEntitlementEnforce(t *testing.T) {
+	entitlementID := "network.none"
```

Comment:
we should use the const we have for this: https://github.com/docker/libentitlement/blob/master/defaults/network-ents.go#L18

Reply:
indeed, will fix for all.